debezium-api/pom.xml

```diff
@@ -15,6 +15,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
The Kafka Connect API is now in the Debezium API library, which is the only change that's somewhat significant. Though I don't know what someone would do with just the Debezium API without the Kafka one.